bundles/framework/publisher2/view/PanelToolLayout.js

```diff
@@ -386,7 +386,13 @@ Oskari.clazz.define('Oskari.mapframework.bundle.publisher2.view.PanelToolLayout'
             if (!elem || this._addedDraggables.includes(elem)) {
                 return;
             }
-            const enabled = tool.state.enabled === true;
+            let enabled = false;
+            if (tool.state) {
+                enabled = tool.state.enabled === true;
+            } else if (typeof tool.handler !== 'undefined') {
+                enabled = true;
+            }
```

Comment:
AbstractPluginTool has isEnabled() that we could use here to make it less hacky: 

https://github.com/oskariorg/oskari-frontend/blob/master/bundles/framework/publisher2/tools/AbstractPluginTool.js#L119-L121

This PR also adds isEnabled() to MapLegend.js LayerTool so we could just implement isEnabled() for other LayerTools and use isEnabled() calls here to get rid of this if/else.